authui/src/authflowv2/icons/material-symbols.css

```diff
@@ -30,4 +30,8 @@
 
   /* Support for IE. */
   font-feature-settings: "liga";
+
+  &.material-icons--filled {
+    font-variation-settings: "FILL" 1;
```

Comment:
I added this class because only the icon in alert message uses filled icon currently.

Reply:
I found this reference https://developers.google.com/fonts/docs/material_symbols#what_are_material_symbols . Please paste this reference link in the source code so that future reader know other than `FILL`, we can also use `wght`, `GRAD`, and `opsz`.